### PR TITLE
Update Sei to v3.9.0

### PR DESCRIPTION
**Pull request by PoolPirate**

### CODE REVIEW

**Comment by PoolPirate** [comment]
Also updated some of the env vars to match what I'm running successfully. The previous ones were still configured for testnet

File: seinetwork/build.yml [hunk 1]
```diff
@@ -5,10 +5,11 @@ services:
     build:
       context: ../
       args:
+        GOLANG_VERSION: 1.20.1
```
Comment (tombeynon) [inline seinetwork/build.yml:8]:
```suggestion
        GOLANG_VERSION: 1.20.1-buster
```

**Review by tombeynon — COMMENTED** [review]
Awesome thanks so much!

File: seinetwork/build.yml (hunk 1 above)
Comment (tombeynon) [reply seinetwork/build.yml:8]: Could we just lock this to buster

**Comment by tombeynon** [comment]
Pre-release image: `ghcr.io/akash-network/cosmos-omnibus:v0.4.15-rc2-seinetwork-v3.9.0`

**Pull request merged**